newsfragments/3998.fixed.md

```diff
@@ -0,0 +1 @@
+allow extraction of `&Bound` in `#[setter]` methods
```

Comment:
I slightly prefer to have Sentence case in these:

```suggestion
Allow extraction of `&Bound` in `#[setter]` methods.
```

(Not a huge deal as I often adjust the wordings of the fragments for consistency upon release. The most important thing is that there's a fragment at all, so that I have something to include 😂)